packages/protocol/src/providers/mainnet/MorphoCompound.sol

```diff
@@ -94,7 +94,7 @@ contract MorphoCompound is ILendingProvider {
     (uint256 ratePerBlock,,) = ILens(LENS).getAverageSupplyRatePerBlock(_getCToken(vault.asset()));
     //no. of blocks per year
     //convert from 1e18 to 1e27
-    rate = ratePerBlock * 2102400 * 10 ** 9;
+    rate = ratePerBlock * 2612670 * 10 ** 9;
```

Comment:
@pedrovalido do we need to update this parameter in all CompoundV2-like providers? If so, would you like to just do it in this PR or do it on a separate branch-PR ?

Reply:
Yes, we do. I've fixed it and committed the changes :)